### PR TITLE
update default behavior of appliesToType

### PR DESCRIPTION
**Pull request by njogz**
Update documentation to better describe the default behavior introduced in this [PR](https://github.com/medic/medic-conf/pull/361)

### CODE REVIEW

File: content/en/apps/reference/contact-page.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Each field that can be shown on a contact's profile is defined as an object in t
 | `translate` | `boolean` | Whether or not to translate the `value`. Defaults to false. | no |
 | `context` | `object` | When `translate: true` and `value` uses [translation variables](https://angular-translate.github.io/docs/#/guide/06_variable-replacement), this value should provide the translation variables. | no |
 | `appliesIf` | `function()` or `boolean` | Return true if the field should be shown. | no |
-| `appliesToType` | `string[]` | Filters the contacts for which `appliesIf` will be evaluated. For example, `['person']` or `['clinic', 'health_center']`. | no |
+| `appliesToType` | `string[]` | Filters the contacts for which `appliesIf` will be evaluated. For example, `['person']` or `['clinic', 'health_center']`. It defaults to all types if it is not defined. | no |
```
Comment (garethbowen) [inline content/en/apps/reference/contact-page.md:42]: Nice! I think we also need to update the Condition Cards documentation on line 56.

**Review by garethbowen — CHANGES_REQUESTED** [review]
Nice! One comment inline.

**Review by garethbowen — APPROVED** [review]
Nice!

**Comment by abbyad** [comment]
Is this PR ready to merge now that https://github.com/medic/medic-conf/issues/164 is closed?

**Pull request merged**